documents/policies/external/Scramble Accountability.md

```diff
@@ -1,91 +1,103 @@
 # WCA Scramble Accountability Policy
 
-### Version 1.0 {.version}
+### Version 2.0 {.version}
 
 ## Purpose
 
-The purpose of this policy is to define standards of how official WCA Scrambles, which in addition to the WCA Regulations, must be treated. The policy is targeted towards the WCA Delegate and organization team. Not fully following the Scramble Accountability Policy may result in disqualification of the affected results, at the discretion of the WCA Regulations Committee.
+The purpose of this policy is to define standards on how official WCA scrambles must be treated. The policy is targeted towards WCA Delegates and organization teams. Not fully following the Scramble Accountability Policy may result in disqualification of the affected results, at the discretion of the WCA Regulations Committee or the WCA Quality Assurance Committee.
```

Comment:
It is preferred to have the ability to disqualify results for WSAP violations to be limited to WRC at the moment. WQAC can separately operate within its power to advise WRC in these matters but whether such a decision is in violation of Regulation 1c1 is within WRC's scope.

Reply:
But we were previously advised that this responsibility may be added for our team. Has the Board changed its stance on this topic?

Reply:
Removed mention of the WQAC